src/core/config/mac.h

```diff
@@ -139,6 +139,16 @@
 #define OPENTHREAD_CONFIG_MAC_MAX_TX_ATTEMPTS_INDIRECT_POLLS 4
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_MAC_CSL_REQUEST_AHEAD_THRESHOLD_US
```

Comment:
Is this really a "threshold"? Is this really just trying to target when the MAC will deliver the CSL frame to sub-MAC?

```suggestion
 * @def OPENTHREAD_CONFIG_MAC_CSL_REQUEST_AHEAD_US
```